packages/kit/src/components/AccountSelector/hooks/useAccountSelectorCreateAddress.tsx

```
@@ -127,11 +127,19 @@ export function useAccountSelectorCreateAddress() {
-          indexedAccountId: account?.indexedAccountId,
-          ...hwUiControlParams,
-        });
+        const result =
+          await serviceBatchCreateAccount.addDefaultNetworkAccounts({
+            walletId: account?.walletId,
+            indexedAccountId: account?.indexedAccountId,
+            ...hwUiControlParams,
+          });
+        if (
+          result &&
+          result?.failedAccounts?.length &&
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)